src/dynamics/mpas/dyn_comp.F90

```diff
@@ -32,15 +51,35 @@ module dyn_comp
 
     public :: dyn_debug_print
     public :: mpas_dynamical_core
+    public :: ncells, ncells_solve, nedges, nedges_solve, nvertices, nvertices_solve, nvertlevels
+    public :: ncells_global, nedges_global, nvertices_global, ncells_max, nedges_max
+    public :: sphere_radius
+    public :: deg_to_rad, rad_to_deg
```

Comment:
I totally agree that this is really suboptimal, but the reason is explained in the commit message of https://github.com/ESCOMP/CAM-SIMA/pull/267/commits/932ef82f6c3d3a283e214b9894facb333f61ed62.

The instance of MPAS dynamical core resides in `dyn_comp`. `dyn_grid` depends on `dyn_comp` because it needs to interact with the instance. So, if `dyn_comp` were to depend on `dyn_grid` for grid variables, there would be a circular dependency issue.

Reply:
Thanks for the explanation!  In that case I think this is fine as-is.